docs/python-api.md

```diff
@@ -1079,12 +1079,22 @@ The following constants are used throughout the `tskit` API.
 
 ### Tree and tree sequence classes
 
+#### {class}`Tree` docs
```

Comment:
How about API instead of docs?

Reply:
Does the {class}`Tree` thing in the title do something different to just "Tree docs"?

Reply:
> Does the {class}Tree thing in the title do something different to just "Tree docs"?

It marks it in bold in the navbar, which I think is visually helpful. But not much otherwise, given it links to the documentation just underneath it.

Reply:
OK, just checking it didn't do something mad with the links.

Reply:
> How about API instead of docs?

Yes, that would work. If so, I wonder if we want to call the whole section "API reference" too, instead of "Reference documentation"

Reply:
Sure, whatever you think

Reply:
Done. Should be OK to merge